client/src/api/pitch/requests.ts

```diff
@@ -0,0 +1,40 @@
+import { AxiosResponse } from 'axios';
+
+import { ErrorWrapper } from '../types';
+import { get, put } from '../builders';
+
+import * as Types from './types';
+
+const PITCH_ENDPOINT = '/pitch';
+
+const getApprovedPitches = async (): Promise<
+  AxiosResponse<Types.GetPitchesResponseType> | ErrorWrapper
+> => await get(`${PITCH_ENDPOINT}?approved=true`, 'GET_PITCH_FAIL');
+
+const getOpenTeams = async (
```

Comment:
You should add comments above each function to describe what they're doing like how the other requests.ts files do. This may be my fault for not writing comments to begin with when I first wrote the pitch api wrapper functions buuuut :))))